### PR TITLE
feat(core/serverGroup): Use flexbox instead of bootstrap grid to layout running tasks

### PR DESCRIPTION
**Pull request by christopherthielen**

### CODE REVIEW

**Comment by christopherthielen** [comment]
Before:

screen shot 2018-03-08 at 4 31 16 pm


After:

screen shot 2018-03-08 at 4 34 46 pm

**Comment by erikmunson** [comment]
this is great. i don't have anything to add

File: app/scripts/modules/core/src/serverGroup/details/RunningTasks.tsx [hunk 1]
```diff
@@ -36,23 +35,18 @@ export class RunningTasks extends React.Component<IRunningTasksProps> {
...
-        </strong>
-      </div>
-    </div>
+  <div>
+    <strong>{props.task.name}</strong>
     {displayableTasks(props.task.steps).map((step, index) => (
-      <div className="row" key={index}>
-        <div className="col-md-7 col-md-offset-0">
-          <span className="small"><StatusGlyph item={step}/></span>{' '}{robotToHuman(step.name)}
+      <div className="flex-container-h baseline margin-between-sm" key={index}>
```
Comment (anotherchrisberry) [inline app/scripts/modules/core/src/serverGroup/details/RunningTasks.tsx]: Where is `margin-between-sm` defined?

Comment (erikmunson) [reply app/scripts/modules/core/src/serverGroup/details/RunningTasks.tsx]: #4978

Comment (anotherchrisberry) [reply app/scripts/modules/core/src/serverGroup/details/RunningTasks.tsx]: 👍

**Pull request merged**